cogs/internallogs.py

```diff
@@ -10,6 +10,25 @@ def __init__(self, bot: Bot):
 
     @commands.Cog.listener()
     async def on_guild_join(self, guild: Guild):
+        cursor = await Utils.mysql_login()
+        database = cursor.cursor()
+        database.execute("SELECT * FROM blacklist WHERE guild_id = %s", [guild.id])
+
+        try:
+            result = database.fetchone()[0]
+            print(f'Guild {guild} attempted to add {self.bot.user.name}, but was blacklisted.')
+            embed = Embed(title="Joined a guild!", color=Utils.Colors.red)
+            embed.add_field(name="Name", value=guild.name, inline=True)
+            embed.add_field(name="ID", value=guild.id, inline=True)
+            embed.description(f'Guild attempted to add {self.bot.user.name}, but is blacklisted.')
+            embed.set_thumbnail(url=guild.icon.url)
+            async with aiohttp.ClientSession() as client_session:
+                webhook = Webhook.from_url(Utils.get_data()['Logs']["JoinWebhook"], session=client_session)
```

Comment:
Yeah maybe we don't wanna use the main join webhook, not really needed to change though